openlibrary/plugins/openlibrary/lists.py

```diff
@@ -641,3 +644,18 @@ def get_active_lists_in_random(limit=20, preload=True):
     lists = f(limit=limit, preload=preload)
     # convert rawdata into models.
     return [web.ctx.site.new(xlist['key'], xlist) for xlist in lists]
+
+def create_list_preview(lst_key):
```

Comment:
I think these two lines
```
  lst = web.ctx.site.get(lst_key)
  five_seeds = lst.seeds[0:5]
```
can probably just be moved into here: https://github.com/internetarchive/openlibrary/pull/6608/files#diff-d031698c982a0845a0ce52eb88254210a01c2ae9f5ab95c45b6049130a2f4861R528

Reply:
Fixed